README.md

```diff
@@ -1,4 +1,355 @@
 # MicroProfile Tutorial (Still Under Construction)
 This tutorial contains 5 sub modules that demonstrate different MicroProfile technologies. Each module can be completed independent of one another by navigating to the required branch with the finished code from the previous module being provided for you. This README will also change depending on the branch you select to give you specific instructions on how to finish each module.
 
-## Module 5
\ No newline at end of file
+## Module 5
+
+### About this module 
+
+This module will show you how to take advantage of the Health and Metrics features added into MicroProfile 1.2 to see if your microservices are responding and provide metric information regarding these services such as CPU load, JVM usage and many other useful information.
+ 
+
+### Before we start
+
+Make sure you have all the prerequisites installed before continuing with these instructions. Clone down this repository
+
+`https://github.com/OpenLiberty/tutorial-microprofile.git`
+
+and navigate to the repository you just cloned and change branch to module 5 tutorial-microprofile
+
+git checkout `module5`
+
+
+### Health
+
+A health check is a MicroProfile Health API implementation that is provided by a microservice. We use health checks to assess the health of a service.
+
+Some basic files have been provided for you such as a basic pom file for building our application with maven, a License file, some basic Open Liberty configuration in the form of xml and the directory structure required for this microservice.
+
+### Assessing health of microservice
+
+1. Navigate into the following directory `microservice-gateway/src/main/java/application/rest/Gateway.java`
+2. We will use this file for listing all the seperate REST API calls we require from our back-end microservices. Firstly add all the required imports needed for this class specified below:
+```
+package application.rest;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.json.JsonObject;
+import javax.inject.Inject;
+```
+
+3. Now we need to add the required REST API calls to the class. We add one for each microservice. We are also using CDI to inject the class Proxy which is responsible for connecting to the back-end:

```

Comment:
This is correct except when I go onto branch (Module 5) I see this code already exists when I do not think it should as the user should be inserting this code on their own?

Reply:
done :)